examples/interpreting_monte_carlo_tree_search.ipynb

```diff
@@ -827,31 +828,27 @@
-      "  warnings.warn(\n",
       "/home/hwpang/miniforge3/envs/chemprop_v2_dev/lib/python3.11/site-packages/lightning/fabric/plugins/environments/slurm.py:204: The `srun` command is available on your system but is not used. HINT: If your intention is to run Lightning on SLURM, prepend your python command with `srun` like so: srun python /home/hwpang/miniforge3/envs/chemprop_v2_dev/lib/pyt ...\n",
       "/home/hwpang/miniforge3/envs/chemprop_v2_dev/lib/python3.11/site-packages/lightning/pytorch/trainer/connectors/data_connector.py:441: The 'predict_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=63` in the `DataLoader` to improve performance.\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 10 11 12 13 14\n",
-      "/home/hwpang/Projects/chemprop_v2_dev/chemprop/chemprop/data/dataloader.py:52: UserWarning: Dropping last batch of size 1 to avoid issues with batch normalization (dataset size = 1, batch_size = 64)\n",
-      "  warnings.warn(\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 11 12 13 14 15\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 8 9 10 11 12\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 7 8 9 10 11\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 1 2 3 4 5\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 0 1 3 4 5\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 0 1 2 3 4\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 11 12 13 14 15\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 8 9 10 11 12\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 7 8 9 10 11\n",
-      "[13:43:00] Can't kekulize mol.  Unkekulized atoms: 10 11 12 13 14\n"
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 10 11 12 13 14\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 11 12 13 14 15\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 8 9 10 11 12\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 7 8 9 10 11\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 1 2 3 4 5\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 0 1 3 4 5\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 0 1 2 3 4\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 11 12 13 14 15\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 8 9 10 11 12\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 7 8 9 10 11\n",
+      "[12:06:08] Can't kekulize mol.  Unkekulized atoms: 10 11 12 13 14\n"
      ]
     },
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "CPU times: user 1min 45s, sys: 554 ms, total: 1min 46s\n",
-      "Wall time: 4.85 s\n"
+      "CPU times: user 2min 44s, sys: 97.7 ms, total: 2min 44s\n",
```

Comment:
Any idea why previously the algorithm took 1.75 minutes, but this time it took 2.75 minutes?

Reply:
I would guess that estes is just more busy when I run the second time. The time shows here is not really meaningful benchmark time because it only measures one single execution, which can be affected by many things, depending on what other things the system is doing at that time. A more rigorous timing benchmark is to execute multiple times and show the statistics. This timing here is only to give the user a head up of how long this cell might take.